### PR TITLE
adding option to swap decks on rematch

### PR DESCRIPTION
**Pull request by SkyJedi**

### CODE REVIEW

File: client/Components/Games/NewGame.jsx [hunk 1]
```diff
@@ -36,7 +36,8 @@ class NewGame extends React.Component {
             password: '',
             useGameTimeLimit: false,
             hideDecklists: false,
-            gameTimeLimit: 35
+            gameTimeLimit: 35,
+            swap: false
```
Comment (cryogen) [inline client/Components/Games/NewGame.jsx:40]: This shouldn't be necessary if you're not offering an option at game creation, just default it on the server

File: client/Components/Games/NewGame.jsx [hunk 2]
```diff
@@ -85,7 +86,8 @@ class NewGame extends React.Component {
             expansions: this.state.expansions,
             useGameTimeLimit: this.state.useGameTimeLimit,
             gameTimeLimit: this.state.gameTimeLimit,
-            hideDecklists: this.state.hideDecklists
+            hideDecklists: this.state.hideDecklists,
+            swap: this.state.swap
```
Comment (cryogen) [inline client/Components/Games/NewGame.jsx:90]: see above, default this server side

**Comment by SkyJedi** [comment]
please merge https://github.com/keyteki/keyteki/pull/1244 first and i'll update this one to include the sexy reversal code

**Comment by cryogen** [comment]
Done

**Comment by SkyJedi** [comment]
this should be good to go

**Pull request merged**